### PR TITLE
add specific userinfo interface

### PR DESCRIPTION
**Pull request by goumang2010**
per https://mp.weixin.qq.com/debug/wxadoc/dev/api/open.html#wxgetuserinfoobject

### CODE REVIEW

**Comment by emeryao** [comment]
thanks for your PR 
great work 👍 @goumang2010

File: we-app.d.ts [hunk 1]
```diff
@@ -1451,9 +1451,26 @@ declare namespace WeApp {
         success?: (res?: UserInfo) => void;
     }
 
+    interface WxUserInfo {
+        /**	用户昵称 */
```
Comment (emeryao) [inline we-app.d.ts]: please remove the spaces at the beginning of each jsdoc comment

File: we-app.d.ts [hunk 2]
```diff
@@ -1451,9 +1451,26 @@ declare namespace WeApp {
         success?: (res?: UserInfo) => void;
     }
 
+    interface WxUserInfo {
+        /**	用户昵称 */
+        nickName: string;
+        /**	用户头像，最后一个数值代表正方形头像大小（有0、46、64、96、132数值可选，0代表640*640正方形头像），用户没有头像时该项为空。若用户更换头像，原有头像URL将失效。 */
```
Comment (emeryao) [inline we-app.d.ts]: please replace all Chinese punctuations(标点符号) to a `space` or *half-size*(半角) punctuations to reduce the file size 😄

**Comment by emeryao** [comment]
@goumang2010 
Can you please make some improvements according to my comments above?

**Comment by emeryao** [comment]
it is much appreciated if you update the date in [README.md](https://github.com/Emeryao/typed-we-app#last-update) and version to `1.6.0-update.3` in [package.json](https://github.com/Emeryao/typed-we-app/blob/master/package.json) 😜

**Comment by emeryao** [comment]
🚪 hey man are you still there ? @goumang2010

**Comment by goumang2010** [comment]
@Emeryao Yes, I'm sorry that there is some problem with my email, so I didn't received the notification.  I'll revise it later. Thank you.

**Comment by goumang2010** [comment]
@Emeryao I have updated the commits, please check it.

**Comment by emeryao** [comment]
PR merged @goumang2010 
Thanks for your great effort 👍

**Pull request merged**